test/setup.js

```diff
@@ -17,6 +17,14 @@ jest.mock('NativeModules', () => {
 });
 jest.mock('NativeEventEmitter');
 
+jest.mock('react-native-device-info', () => {
+    return {
+        getVersion: () => '0.0.0',
+        getBuildNumber: () => '0',
+        getModel: () => 'iPhone X',
```

Comment:
Does it make sense for tests to default to iPhone X? I don't know what our usage numbers are between the two